onnxmltools/convert/main.py

```diff
@@ -8,12 +8,12 @@
 from .common import utils
 
 
-def convert_sklearn(model, name=None, initial_types=None, doc_string='', targeted_onnx=onnx.__version__):
+def convert_sklearn(model, name=None, initial_types=None, doc_string='', targeted_onnx=onnx.__version__, label_variable_name=None, probabilities_variable_name=None):
```

Comment:
any user case or particular reason to make this API change?

Reply:
PredictionUnit API in Windows 10 requires specific output name "Probability" instead of "probabilities". I am open to nicer design changes but this was easiest fix.

Reply:
This is a special requirement, which can't be adopted for a general API design.

Why not directly modify the output name after the model converted.

Reply:
I am not adding anything about my special requirements. What I added is capability to configure hard-coded output names. However, it is good a suggestion to do it outside. How can I rename output node after conversion?

Reply:
It's a bit strange to specify label_variable_name or probabilities_variable_name. There is no formal definitions for them! Let's think about an ensemble model, a pipeline with one linear classifier and random forest classifier. Which variable should be associated with label_variable_name/probabilities_variable_name?

A basic idea for modifying ONNX model in python (assume that your model variable is `model`: 
  1. Iterate through `model.graph.node` and change all the names of being targeted. 
  2. Also, modify names in `model.output` accordingly